tests/python/relay/aot/aot_test_utils.py

```diff
@@ -772,7 +781,16 @@ def compile_and_run(
     use_runtime_executor: bool = True,
     target: str = "c",
     target_opts: Dict = None,
+    test_dir: str = None,
+    verbose: bool = False,
 ):
+    """
+    Compiles the AOTTestModel and runs using AOT test runner
+    Parameters
+    ----------
+    test_dir : This path will contain build, codegen, include directories
```

Comment:
Shall we use numpy style docstrings? Please refer to other python code in the codebase.